build-system/tasks/check-sourcemaps.js

```diff
@@ -164,7 +165,7 @@ function checkSourcemapMappings(sourcemapJson) {
     log(helpMessage);
     throwError('Found mapping for incorrect code');
   }
-  if (generatedCodeColumn != 0 || sourceCodeColumn != 0) {
+  if (generatedCodeColumn != expectedColStart || sourceCodeColumn != 0) {
     log(red('ERROR:'), 'Found mapping for incorrect (non-zero) column.');
```

Comment:
This comment is no longer correct. I think it's enough to drop the `(non-zero)`.